index.js

```diff
@@ -167,7 +167,8 @@ class Ky {
 		this._response = this._fetch();
 
 		for (const type of responseTypes) {
-			this._response[type] = this._retry(async () => {
+			const isRetry = retryMethods.has(this._options.method.toLowerCase());
```

Comment:
isRetry might be a little confusing since it's more like a "allowRetry" check

Reply:
Maybe `isRetriableMethod`?